### PR TITLE
Release 0.3.0 changes

### PR DESCRIPTION
**Pull request by deathbots**
- Added autogenerated HTML project documentation which publishes to https://github.com/runZeroInc/runzero-sdk-py in cadence with releases.
- Added task, scan, and template management capabilities to SDK.
- Added explorer management capabilities to SDK.
- Added hosted zone enumeration to SDK.
- Added source_id field to Task objects.
- Updated example code to demonstrate forcing data to be merged onto a known runZero asset ID, allowing for arbitrary asset matching, programmatically defined.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -6,6 +6,7 @@ license = "BSD-2-Clause"
 authors = ["runZero <support@runzero.com>"]
 readme = "README.md"
 homepage = "https://runzero.com/"
+documentation = "https://runzeroinc.github.io/runzero-sdk-py"
```
Comment (treyburn) [inline pyproject.toml:9]: We are missing the version bump above this.

File: .githooks/pre-push [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+# $1 -- Name of the remote to which the push is being done
+# $2 -- URL to which the push is being done
+#
+# If pushing without using a named remote those arguments will be equal.
+#
+# Information about the commits which are being pushed is supplied as lines to
+# the standard input in the form:
+#
+#   <local ref> <local oid> <remote ref> <remote oid>
+#
```
Comment (treyburn) [inline .githooks/pre-push:14]: Did we mean to include the githook in the public repo?

Comment (deathbots) [reply .githooks/pre-push:14]: Yes, for the willing. The _daring_ even :) At least it's optional.

File: pyproject.toml (hunk 1 above)
Comment (deathbots) [reply pyproject.toml:9]: Not yet, that comes in the next PR!

**Review by treyburn — APPROVED** [review]

**Pull request merged**